core/pom.xml

```diff
@@ -116,6 +116,10 @@
       <groupId>org.reactivestreams</groupId>
       <artifactId>reactive-streams</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-net</groupId>
+      <artifactId>commons-net</artifactId>
+    </dependency>
```

Comment:
What is the strategy around adding dependencies? Is that fine to have it here?

Reply:
I think if we add any dependencies for this, we should make them optional, as I expect folks will commonly not use an address translator, so we should instead have users who desire to use this feature add this as a dependency in their project (which we could document the requirement for somewhere).

Also I think it would be nice to consider using https://github.com/seancfoley/IPAddress instead, as [C* also uses this](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/config/SubnetGroups.java) and it carries the benefits of having no other dependencies and it also supports IPv6.

Reply:
Changed to IPAddress and made it optional.